src/twisted/words/test/test_irc.py

```diff
@@ -9,13 +9,27 @@
 import operator
 import time
 
+from twisted.internet import protocol, task
+from twisted.python.filepath import FilePath
+from twisted.python.compat import unicode
+from twisted.test.proto_helpers import StringTransport, StringIOWithoutClosing
 from twisted.trial import unittest
 from twisted.trial.unittest import TestCase
 from twisted.words.protocols import irc
 from twisted.words.protocols.irc import IRCClient, attributes as A
-from twisted.internet import protocol, task
-from twisted.test.proto_helpers import StringTransport, StringIOWithoutClosing
-from twisted.python.filepath import FilePath
+
+
+
+def assertEqualBufferValue(bufferValue, val):
```

Comment:
As per https://twisted.readthedocs.io/en/twisted-16.4.1/core/development/policy/coding-standard.html#docstrings - "all methods, functions, classes, and modules must have docstrings".  So this should have a docstring :).  The docstring should describe its behavior, the type and purpose of `bufferVal` and `val` parameters.